static/js/publisher/pages/Releases/api/releases.js

```diff
@@ -21,8 +21,8 @@ export function fetchReleases(onComplete, releases, csrfToken, snapName) {
   return queue;
 }
 
-export function fetchReleasesHistory(csrfToken, snapName) {
-  return fetch(`/${snapName}/releases/json`, {
+export function fetchSnapReleaseStatus(csrfToken, snapName) {
```

Comment:
This could probably just use `window.CSRF_TOKEN`

Reply:
I'll do a quick follow-up to this PR for this :+1: